index.html

```diff
@@ -609,8 +634,18 @@ <h2>Queue a <code>PerformanceEntry</code></h2>
             </li>
           </ol>
         </li>
-        <li>If the <var>add to performance entry buffer</var> flag is set, add
-        <var>new entry</var> to the <a>performance entry buffer</a>.
+        <li>
+          Determine if you should add <var>new entry</var> to a
+          <a>performance entry buffer</a>:
+          <ol>
+            <li> Let <var>tuple</var> be the <a>relevant performance entry
```

Comment:
Nit: Spurious space, missing period at end of line.

Reply:
Done.